documentation/sphinx/source/authorization.rst

```diff
@@ -0,0 +1,119 @@
+   Claim           ``sub``     No       Subject of the token.                                `RFC7519 Section 4.1.2 <https://www.rfc-editor.org/rfc/rfc7519#section-4.1.2>`_
+   Claim           ``aud``     No       Intended recipients of the token. Must be an array.  `RFC7519 Section 4.1.3 <https://www.rfc-editor.org/rfc/rfc7519#section-4.1.3>`_
+   Claim           ``jti``     No       String that uniquely identifies a token.             `RFC7519 Section 4.1.7 <https://www.rfc-editor.org/rfc/rfc7519#section-4.1.7>`_
+   =============== =========== ======== ==================================================== ================================================================================
+
+Public keys with which to verify the token must be serialized in `JWK Set <https://www.rfc-editor.org/rfc/rfc7517#section-5>`_ format and stored in a file.
+The location of the key set file must be passed as command line argument ``--authorization-public-key-file`` to the ``fdbserver`` executable.
+Public keys in the set must be either `RSA <https://datatracker.ietf.org/doc/html/rfc7518#section-6.3>`_ public keys
+containing ``n`` and ``e`` parameters, each containing `Base64urlUInt <https://www.rfc-editor.org/rfc/rfc7518#section-2>`_-encoded modulus and exponent,
+or `Elliptic Curve <https://datatracker.ietf.org/doc/html/rfc7518#section-6.2>`_ public keys on a ``P-256`` curve,
+where ``crv`` parameter is set to ``P-256`` and ``x`` and ``y`` parameters contain
+`base64url <https://datatracker.ietf.org/doc/html/rfc4648#section-5>`_-encoded affine coordinates.
+In addition, each public key JSON object in set must contain ``kty`` (set to either ``EC`` or ``RSA``) field to indicate public key algorithm,
+along with ``kid``, and ``alg`` fields to be compared against their token header counterparts.
+Private keys are strongly recommended against being included in the public key set and, if found, are excluded from consideration.
+
+.. note:: By design, FoundationDB authorization feature does not support revocation of outstanding tokens.
+          Use extra caution in signing tokens with long token durations.
+
+Enabling Clients to use Authorization Tokens
+============================================
+
+In order to use an untrusted client with an authorization token, a client must be configured to trust the server's CA,
+but must not be configured to use the client's own certificates and keys.
+More concretely, the client's ``TLS_CA_FILE`` must include the server's root CA certificate,
+but the client must not be configured with its own ``TLS_CERTIFICATE_FILE`` or ``TLS_KEY_FILE``, neither programmatically nor by environment variable.
+Before performing a tenant data read or update, a client must set transaction option ``AUTHORIZATION_TOKEN`` with the token string as argument.
+It is the client's responsibility to keep the token up-to-date, by timely assigning a new token to the transaction object.
```

Comment:
Maybe it would be useful for people to mention that this is the same setup that a web browser would use? Also maybe mention explicitly that the client will authenticate the FDB servers (so that it becomes clear what the benefits of TLS (apart from encryption) are.

Reply:
Good thinking. I'll comment on the similarity as a side note.